<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 7.0.1 (Python 3.9.13 on win32)" generated="2025-02-09T19:36:50.106859" rpa="false" schemaversion="5">
<suite id="s1" name="Robot Interactive Console" source="c:\Users\few66\.vscode\extensions\robocorp.robotframework-lsp-1.13.0\src\robotframework_ls\vendored\robotframework_interactive\robot_interactive_console.robot">
<test id="s1-t1" name="Default Task/Test" line="5">
<kw name="Interpreter Main Loop" owner="MainLoop">
<status status="PASS" start="2025-02-09T19:36:50.126526" elapsed="6.049150"/>
</kw>
<status status="PASS" start="2025-02-09T19:36:50.125528" elapsed="6.050148"/>
</test>
<status status="PASS" start="2025-02-09T19:36:50.107862" elapsed="6.068813"/>
</suite>
<statistics>
<total>
<stat pass="1" fail="0" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="1" fail="0" skip="0" id="s1" name="Robot Interactive Console">Robot Interactive Console</stat>
</suite>
</statistics>
<errors>
<msg time="2025-02-09T19:36:50.101445" level="WARN">Error in file 'c:\Users\few66\.vscode\extensions\robocorp.robotframework-lsp-1.13.0\src\robotframework_ls\vendored\robotframework_interactive\robot_interactive_console.robot' on line 4: Singular section headers like '*** Test Case ***' are deprecated. Use plural format like '*** Test Cases ***' instead.</msg>
</errors>
</robot>
